fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageSessionTest.java

```diff
@@ -88,7 +88,7 @@
  *
  * @author dbernstein
  */
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
It would appear that the use of `Silent` is not recommended. Do we have a good reason here?
```
     * Engineers should care for removing unused stubbings because they are dead code,
     * they add unnecessary details, potentially making the test code harder to comprehend.
     * If you have good reasons to use the silent runner, let us know at the mailing list
     * or raise an issue in our issue tracker.
     * The purpose of silent implementation is to satisfy edge/unanticipated use cases,
     * and to offer users an opt-out.
     * Mockito framework is opinionated to drive clean tests but it is not dogmatic.
```

Reply:
I'm not sure why that was done. I can try running the tests without it and see what happens.

Reply:
Didn't seem to be doing anything -- removed.